src/components/validation/resources/mod.rs

```diff
@@ -142,6 +142,8 @@ fn deserializer_config_to_serializer(config: &DeserializerConfig) -> encoding::S
         // `message` field... but it's close enough for now.
         DeserializerConfig::Bytes => SerializerConfig::Text(TextSerializerConfig::default()),
         DeserializerConfig::Json { .. } => SerializerConfig::Json(JsonSerializerConfig::default()),
+        // TODO: how to handle this?
+        DeserializerConfig::Protobuf(_) => todo!(),
```

Comment:
some of the code within this file also uses `todo!`. should I change it to `panic!` anyway?

Reply:
I would prefer to see either `unimplemented!` or `unreachable!` instead of just `todo!` to better document the intent here, unless you _do_ mean to complete this before considering this PR "done".